roles/keystone/templates/etc/keystone/policy.json

```diff
@@ -2,9 +2,11 @@
 
 {
 {% if policy_for_horizon|default('False')|bool %}
-    "admin_required": "role:admin or is_admin:1 or role:cloud_admin or role:project_admin or rule:owner",
+    "admin_required_limit": "role:admin or is_admin:1 or role:cloud_admin or role:project_admin or rule:owner",
```

Comment:
So I verified that the change to the keystone policy.json is not required. You need to make sure that horizon customization is enabled as well as jellyroll. I could not test the actual live migration as I only had one compute host in my env. I was able to start the live migration and the api was invoked correctly.